### DISCUSSION TITLE
Joystick operations

### DISCUSSION THREAD

**Discussion by markusbuchholz** [Robotic-Decision-Making-Lab/blue#159] [Q&A]
Hi @evan-palmer , I wonder if the joystick, which can be launched by setting the joy flag to true, needs to be connected to the topic 'cmd_vel' or if it can be launched and should work as the following command ?:
```bash
ros2 launch blue_bringup bluerov2_heavy.launch.py use_sim:=true use_joy:=true
```
Thanks!

**Comment by evan-palmer** [comment]
Hey @markusbuchholz! I'm working on putting together a tutorial on teleoperation [here](https://robotic-decision-making-lab.github.io/blue/tutorials/teleop). My plan is to have that ready for you tomorrow!

**Comment by evan-palmer** [comment]
Alright, [here](https://robotic-decision-making-lab.github.io/blue/tutorials/teleop) is most of the tutorial. It does reference the new changes that I am wrapping up on [33-feature-ros2-control](https://github.com/Robotic-Decision-Making-Lab/blue/tree/33-feature-ros2-control), but I think it should answer most of your questions.

**Comment by markusbuchholz** [comment]
Hei @evan-palmer,
I appreciate your effort in providing such detailed information on how to manage the joystick. 

I would definitely recommend everyone to use your framework since it works out of the box! 

It is very inspiring and excellently done!

PS. As for my purpose, I map ROS topics (functionality) directly to Mavlink as I have to support also other features and BlueBoat. 

Thanks!

**Comment by evan-palmer** [comment]
Thank you for your kind words Markus! If you are interested in submitting a PR with support for the BlueBoat that would be an awesome contribution!

**Comment by markusbuchholz** [comment]
Yes @evan-palmer , I am interested because I currently work with "tandem" BlueRov-BlueBoat. I cannot promise when I will contribute, but I work with the case on a regular basis.
